### PR TITLE
Add Awards Section with Storyblok. Refactor Categories Interface. add…

### PR DESCRIPTION
**Pull request by AngeloVaca**
… CardSectionStyle2

### CODE REVIEW

**Review by MaxCar31 — APPROVED** [review]
Perfect !

File: src/storyblok/CardSection2SB.astro
Comment (juanfcarrillo) [inline src/storyblok/CardSection2SB.astro:1]: Please add more descriptive components, we could want to reuse them in other page than home

File: src/storyblok/CompetitionSB.astro
Comment (juanfcarrillo) [inline src/storyblok/CompetitionSB.astro:1]: This image field is being deprecated in storyblok cms, try to change it to assets field type

Comment (juanfcarrillo) [inline src/storyblok/CompetitionSB.astro:1]: Please add more descriptive components, we could want to reuse them in other page than home. Competition could be only present in home

**Review by juanfcarrillo — COMMENTED** [review]
Good work team :3 left few comments in order to improve the reuse and naming of the project components

**Pull request merged**